File: WPProekt/Models/Comment.cs
using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations;
using System.Linq;
using System.Web;

namespace WPProekt.Models {
    public class Comment {
        public int ID { get; set; }

        [Required]
        public string AuthorName { get; set; }

        public string Content { get; set; }

        public DateTime DateCreated { get; set; }

        public int PostID { get; set; }

        public Comment() {
            DateCreated = DateTime.UtcNow;
        }
    }
}

File: wp-proekt/src/main/java/finki/dabod/wpproekt/models/Comment.java
package finki.dabod.wpproekt.models;

import lombok.Data;

import javax.persistence.*;
import java.util.Date;

@Data
@Entity
@Table(name = "Comments")
public class Comment {
    @Id
    @GeneratedValue(strategy = GenerationType.AUTO)
    private int id;

    private String authorName;
    private String content;

    private Date dateCreated;

    @ManyToOne(targetEntity = Post.class, fetch = FetchType.LAZY)
    private int postId;

    public Comment(){
        dateCreated = new Date();
    }
}
